komodo/cli.py

```diff
@@ -88,13 +88,16 @@ def _main(args):
 
     create_enable_scripts(komodo_prefix=tmp_prefix, komodo_release=args.release)
 
-    with open(args.locations_config, mode="r", encoding="utf-8") as defs, open(
-        Path(args.release) / "local", mode="w", encoding="utf-8"
-    ) as local_activator, open(
-        Path(args.release) / "local.csh", mode="w", encoding="utf-8"
-    ) as local_csh_activator:
-        defs = yml.safe_load(defs)
-        local.write_local_activators(data, defs, local_activator, local_csh_activator)
+    if args.locations_config is not None:
+        with open(args.locations_config, mode="r", encoding="utf-8") as defs, open(
```

Comment:
This  `with` construct is just so hard to read, when all it is doing is to read one file and write two others. But maybe a rewrite can be done in a different PR.

Reply:
I agree... TBH I didn't dare touch it and didn't want to dig into `komodo.local`. I made an issue for it.